### DISCUSSION TITLE
Making emojis searchable in Obsidian status tags

### DISCUSSION THREAD

**Discussion by danichu1** [BryanJenksCommunity/FAQ#190] [Q&A]
Hello everyone

I tried finding the answer to this question myself, but I wasn't able to find an explanation. In short, there are two things I'd like to achieve. 
1) Visualize a tag as an emoji that is searchable by using words.
2) Future proofing my vault by making sure the system doesn't depend on one plugin or something

Btw, do you use a plugin for this?

Thanks for the help

**Comment by tallguyjenks** [comment]
im not sure what you're trying to accomplish in your first point, in your second point its not hard to do given that its all plain text but at some point you need the plugins or else why bother even using the app, could just use notepad at that point.

do i use a plugin for what?
